src/main/java/bowling/model/Bowling.java

```diff
@@ -0,0 +1,84 @@
+package bowling.model;
+
+import bowling.model.frame.FrameNumber;
+import bowling.model.frame.Frames;
+import bowling.utility.Assert;
+
+import java.util.Objects;
+
+public final class Bowling {
+
+    private final Participant participant;
+    private final Frames frames;
+
+    private Bowling(Participant participant, Frames frames) {
+        Assert.notNull(participant, "participant must not be null");
+        Assert.notNull(frames, "frames must not be null");
+        this.participant = participant;
+        this.frames = frames;
+    }
+
+    public static Bowling from(Participant participant) {
+        return new Bowling(participant, Frames.init());
+    }
+
+    public static Bowling of(Participant participant, Frames frames) {
+        return new Bowling(participant, frames);
+    }
+
+    public boolean isNotFinished() {
+        return !frames.isFinished();
+    }
+
+    public Bowling pitch(Pins countOfHit) {
+        validateState();
+        return of(participant, frames.addedFrames(countOfHit));
+    }
+
+    public Participant participant() {
+        return participant;
+    }
+
+    public Frames frames() {
+        return frames;
+    }
+
+    public FrameNumber nextFrameNumber() {
+        return frames.nextFrameNumber();
+    }
+
+    private void validateState() {
+        if (isFinished()) {
+            throw new IllegalStateException(String.format("bowling(%s) is already finished", this));
+        }
+    }
+
+    private boolean isFinished() {
+        return !isNotFinished();
```

Comment:
```suggestion
        return frames.isFinished();
```
부정의 부정보다는 그냥 isFinished를 사용하는게 더 가독성이 좋다고 생각드는데 
어떻게 생각하시나요? 🤔

Reply:
역할의 응집도를 위해 기존 메소드를 활용했는데 방향이 잘못된 것 같습니다. 😅  
이 부분은 수정하겠습니다!! 감사합니다!